configuration/cardano/mainnet-config.json

```diff
@@ -41,6 +41,7 @@
   "TracingVerbosity": "NormalVerbosity",
   "TurnOnLogMetrics": true,
   "TurnOnLogging": true,
+  "ViewMode": "SimpleView",
```

Comment:
I thought the `LiveView` had been removed and there's no reason anymore to specify the `ViewMode`? Similarly for the other changes. cc: @denisshevchenko

Reply:
This was me trying to sync config with what is provided by CI, but it seems I synced in the wrong direction.